apps/web/src/design-system/table/Table.tsx

```diff
@@ -109,10 +109,12 @@ export function Table({
 
       <MantineTable className={classes.root} {...defaultDesign} {...getTableProps()} {...props}>
         <thead>
-          {headerGroups.map((headerGroup) => (
-            <tr {...headerGroup.getHeaderGroupProps()}>
+          {headerGroups.map((headerGroup, i) => (
+            <tr key={i} {...headerGroup.getHeaderGroupProps()}>
               {headerGroup.headers.map((column) => (
-                <th {...column.getHeaderProps()}>{column.render('Header')}</th>
+                <th key={column.Header} {...column.getHeaderProps()}>
```

Comment:
The `Header` field on the column is optional and it can be a string or the function that returns the JSX. 
The same as the above, the keys are auto-generated.

Reply:
Fixed